.github/workflows/ci.yml

```diff
@@ -35,7 +35,22 @@ jobs:
       - test-web-app
       - test-wasm
     runs-on: ubuntu-latest
+    if: always()
     steps:
+      - name: Debug needs
+        env:
+          NEEDS: ${{ toJSON(needs) }}
+        run: printenv NEEDS
+
+      - run: |
+          echo "Oh No, we have jobs that have failed/cancelled/skipped :("
+          exit 42
+        if: >-
+          contains(needs.*.result, 'failure')
+          || contains(needs.*.result, 'skipped')
```

Comment:
When we don't modify rust, the rust job is still executed (but does nothing). So this check on skipped is not affected, right ?

Reply:
Yes, you're right. The jobs is always executed but will performed a greatly reduced amount of work